### PR TITLE
i18n(es): add integrations guide

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- [ ] Minor content fixes (broken links, typos, etc.)
- [ ] New or updated content
- [x] Translated content
- [ ] Changes to the docs site code
- [ ] Something else!

#### Description

- adds `guides/integrations-guide.md` page in Spanish
- page is updated with changes in #761

### CODE REVIEW

File: src/pages/es/guides/integrations-guide.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+- Agregue nuevas funcionalidades a su proyecto, como la generación automática de sitemaps.
+- Escriba código personalizado que se conecte con el proceso de compilación, el entorno de desarrollo y más.
+
+:::note[Estado experimental]
+Solo las integraciones oficiales (publicadas con el prefijo `@astrojs/` en npm) son actualmente compatibles por defecto para proteger a los usuarios de cambios en la API.
+
+**Para habilitar integraciones de terceros:** Ejecute Astro con la extensión `--experimental-integrations`, o incluya `experimental: {integrations: true}` en su archivo de configuración de Astro.
+:::
+
+## Tutorial: Agrega React a su proyecto
```
Comment (agustinmulet) [inline src/pages/es/guides/integrations-guide.md:22]:
```suggestion
## Tutorial: Agregue React a su proyecto
```

File: src/pages/es/guides/integrations-guide.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+
+En este ejemplo, usaremos la integración `@astrojs/react` para hacer que React sea compatible con su proyecto de Astro. El proceso para agregar cualquier otro framework (Preact, Vue, Svelte o Solid.js) es idéntico y se pueden usar los mismos pasos que se describen a continuación.
+
+:::tip[Método rápido]
+¡Astro ofrece el comando `astro add` para automatizar este proceso para todas las integraciones oficiales! En lugar de seguir los siguientes pasos, puedes ejecutar `npx astro add react`. ¡Eso es todo!
+
+Vaya a [configuración de integración automática](/es/guides/integrations-guide/#configuración-de-integración-automática) para obtener más detalles.
+:::
+
+Primero, deberá instalar tanto la integración como cualquier paquete relacionado al paquete que desee usar en su proyecto. Para React, eso significa instalar la integraciones `@astrojs/react` ***y*** los paquetes `react` + `react-dom`.
```
Comment (agustinmulet) [inline src/pages/es/guides/integrations-guide.md:32]:
```suggestion
Primero, deberá instalar tanto la integración como cualquier paquete relacionado al paquete que desee usar en su proyecto. Para React, eso significa instalar las integraciones `@astrojs/react` ***y*** los paquetes `react` + `react-dom`.
```

File: src/pages/es/guides/integrations-guide.md [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+npx astro add react tailwind partytown
+# Usando Yarn
+yarn astro add react tailwind partytown
+# Usando PNPM
+pnpx astro add react tailwind partytown
+```
+
+## Manejando las dependencias de integraciones
+
+Al instalar una integración de Astro en su proyecto, esté atento a las advertencias de "missing peer dependencies" que ve durante el proceso de instalación. No todos los gestores de paquetes instalarán automáticamente las peer-dependencias. Si está en Node v16+ y usa npm, no debería preocuparse por esta sección.
```
Comment (agustinmulet) [inline src/pages/es/guides/integrations-guide.md:93]:
```suggestion
Al instalar una integración de Astro en su proyecto, esté atento a las advertencias de "missing peer dependencies" que ve durante el proceso de instalación. No todos los gestores de paquetes instalarán automáticamente las peer-dependencies. Si está en Node v16+ y usa npm, no debería preocuparse por esta sección.
```

File: src/pages/es/guides/integrations-guide.md [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+
+```diff
+# Ejemplo: Instale las integraciones y frameworks juntos
+- npm install @astrojs/react
++ npm install @astrojs/react react react-dom
+```
+
+Si omite este paso, no se preocupe, Astro le advertirá al iniciar su entorno de desarrollo si se requieren dependencias que no se encuentran en su proyecto.
+
+Administrar sus propias peer-dependencias puede ser un poco más laborioso, pero también le permite controlar exactamente qué versiones de paquetes usa para cosas como React, Tailwind y más. Esto le da más control sobre su proyecto.
```
Comment (agustinmulet) [inline src/pages/es/guides/integrations-guide.md:103]:
```suggestion
Administrar sus propias peer-dependencies puede ser un poco más laborioso, pero también le permite controlar exactamente qué versiones de paquetes usa para cosas como React, Tailwind y más. Esto le da más control sobre su proyecto.
```

File: src/pages/es/guides/integrations-guide.md [hunk 5]
```diff
@@ -0,0 +1,155 @@
...
+
+En el futuro, el comando `astro add` podrá manejar toda esta configuración por usted e instalar automáticamente las peer-dependencias correctas para sus integraciones.
```
Comment (agustinmulet) [inline src/pages/es/guides/integrations-guide.md:105]:
```suggestion
En el futuro, el comando `astro add` podrá manejar toda esta configuración por usted e instalar automáticamente las peer-dependencies correctas para sus integraciones.
```

File: src/pages/es/guides/integrations-guide.md [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+
+```js
+// astro.config.mjs
+import {defineConfig} from 'astro/config';
+import installedIntegration from '@astrojs/vue';
+import localIntegration from './my-integration.js';
+
+export default defineConfig({
+  integrations: [
+    // 1. Imported from an installed npm package
```
Comment (agustinmulet) [inline src/pages/es/guides/integrations-guide.md:128]:
```suggestion
    // 1. Importado desde un paquete npm instalado
```

File: src/pages/es/guides/integrations-guide.md [hunk 7]
```diff
@@ -0,0 +1,155 @@
...
+    installedIntegration(),
+    // 2. Imported from a local JS file
```
Comment (agustinmulet) [inline src/pages/es/guides/integrations-guide.md:130]:
```suggestion
    // 2. Importado desde un archivo JS local
```

File: src/pages/es/guides/integrations-guide.md [hunk 8]
```diff
@@ -0,0 +1,155 @@
...
+    localIntegration(),
+    // 3. An inline object
```
Comment (agustinmulet) [inline src/pages/es/guides/integrations-guide.md:132]:
```suggestion
    // 3. Un objeto inline
```

File: src/pages/es/guides/integrations-guide.md [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+    {name: 'namespace:id', hooks: { /* ... */ }},
+  ]
+})
+```
+
+Consulte la referencia de [API de integración](/es/reference/integrations-reference/) para conocer las diferentes formas para escribir una integración.
+
+### Opciones personalizadas
+
+Las integraciones casi siempre se crean como funciones de fábrica que devuelven el objeto de integración real. Esto le permite pasar argumentos y opciones a la función de fábrica que personaliza la integración para su proyecto.
```
Comment (agustinmulet) [inline src/pages/es/guides/integrations-guide.md:142]: I don't find "función de fábrica" like natural language but I cannot think of another thing to use here 🤔 maybe we could leave "factory function"? Or maybe "función factory"? What do you think?

**Review by agustinmulet — COMMENTED** [review]
Mostly typos and a doubt to discuss 😄

File: src/pages/es/guides/integrations-guide.md (hunk 9 above)
Comment (kevinzunigacuellar) [reply src/pages/es/guides/integrations-guide.md:142]: creo que mantenerla en ingles seria lo mejor

**Comment by kevinzunigacuellar** [comment]
LGTM! 🚀

**Comment by Yan-Thomas** [comment]
Nice to see so many pages being translated!! :rocket:

**Pull request merged**